common/k8s/src/event_source.rs

```diff
@@ -276,17 +276,18 @@ impl K8sEventStream {
                 if let (pod_gen, Some(pod_started_at), Some(pod_name)) =
                     // get pod generation, it will be there if there is an update strategy
                     (
-                    p.metadata
-                        .labels
-                        .as_ref()
-                        .map(|l| {
-                            l.get("pod-template-generation")
-                                .and_then(|g| g.parse::<u64>().ok())
-                        })
-                        .flatten(),
-                    get_pod_started_at(p),
-                    p.metadata.name.as_ref(),
-                ) {
+                        p.metadata
```

Comment:
cargo fmt's preferences have changed apparently

Reply:
yeah, that line have been driving me crazy